metron-platform/metron-indexing/src/test/java/org/apache/metron/indexing/dao/SearchIntegrationTest.java

```diff
@@ -443,11 +495,11 @@ public void all_query_returns_all_results() throws Exception {
     Assert.assertEquals(10, results.size());
     for(int i = 0;i < 5;++i) {
       Assert.assertEquals("snort", results.get(i).getSource().get("source:type"));
-      Assert.assertEquals(10 - i, results.get(i).getSource().get("timestamp"));
+      Assert.assertEquals(10 - i + "", results.get(i).getSource().get("timestamp").toString());
```

Comment:
Why are all the timestamps strings now?

Reply:
The ES and Solr clients return different types (Int vs Longs) so I am converting to string to account for that.  I'm not sure this is the correct approach and open to ideas.